app/ui/nav/navbar.tsx

```diff
@@ -45,9 +45,13 @@ export default function Navbar() {
   let router = useRouter()
   const pathname = usePathname()
   const [current, setCurrent] = useState(pathname)
-
+  const [NETWORK, setNetwork] = useState('')
+  const [WEBSITE, setWebsite] = useState('')
```

Comment:
Please note the difference in the way these variables are accessed here. Accessing them via the constants would lead to a mismatch between the client and the server. This method is only needed for variables that would typically be accessed by the browser i.e with the NEXT_PUBLIC prefix the rest can remain as is.